common/network_manager.go

```diff
@@ -92,20 +87,15 @@ func NewNetworkManager(
 		ctx:              ctx,
 		// TODO: Pass an internal logger instead of basing it on k6's logger?
 		// See https://github.com/grafana/xk6-browser/issues/54
-		logger:                         NewLogger(ctx, state.Logger, false, nil),
-		session:                        session,
-		parent:                         parent,
-		frameManager:                   manager,
-		resolver:                       resolver,
-		credentials:                    nil,
-		reqIDToRequest:                 make(map[network.RequestID]*Request),
-		reqsMu:                         sync.RWMutex{},
-		attemptedAuth:                  make(map[fetch.RequestID]bool),
-		extraHTTPHeaders:               make(map[string]string),
-		offline:                        false,
-		userCacheDisabled:              false,
-		userReqInterceptionEnabled:     false,
-		protocolReqInterceptionEnabled: false,
+		logger:           NewLogger(ctx, state.Logger, false, nil),
+		session:          s,
+		parent:           parent,
+		frameManager:     fm,
+		resolver:         resolver,
+		reqIDToRequest:   make(map[network.RequestID]*Request),
+		reqsMu:           sync.RWMutex{},
```

Comment:
While we're removing things, `reqsMu` also doesn't need initialization. :)

Reply:
Definitely :)